add govkbot

nikepan opened the pull request:
Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard) before sending pull request. Thanks!

**Please provide package links to:**
- github.com repo: https://github.com/nikepan/govkbot
- godoc.org: https://godoc.org/github.com/nikepan/govkbot
- goreportcard.com: https://goreportcard.com/report/github.com/nikepan/govkbot
- coverage service link (gocover, coveralls etc.): https://gocover.io/github.com/nikepan/govkbot


**Note**: that new categories can be added only when there are 3 packages or more.

**Make sure that you've checked the boxes below before you submit PR:**
- [x] I have added my package in alphabetical order
- [x] I know that this package was not listed before
- [x] I have added godoc link to the repo and to my pull request
- [x] I have added coverage service link to the repo and to my pull request
- [x] I have added goreportcard link to the repo and to my pull request
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

Thanks for your PR, you're awesome! :+1:

felipeweb commented:
@nikepan please read https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md and follow https://github.com/avelino/awesome-go/blob/master/.github/PULL_REQUEST_TEMPLATE.md

nikepan commented:
Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard) before sending pull request. Thanks!

**Please provide package links to:**
- github.com repo: https://github.com/nikepan/govkbot
- godoc.org: https://godoc.org/github.com/nikepan/govkbot
- goreportcard.com: https://goreportcard.com/report/github.com/nikepan/govkbot
- coverage service link (gocover, coveralls etc.): 


**Note**: that new categories can be added only when there are 3 packages or more.

**Make sure that you've checked the boxes below before you submit PR:**
- [x] I have added my package in alphabetical order
- [x] I know that this package was not listed before
- [x] I have added godoc link to the repo and to my pull request
- [x] I have added coverage service link to the repo and to my pull request
- [x] I have added goreportcard link to the repo and to my pull request
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

Thanks for your PR, you're awesome! :+1:

felipeweb commented:
@nikepan  you need improve coverage

nikepan commented:
coverage fixed
